src/app/component/eco-news/create-news/create-news.component.html

```diff
@@ -40,6 +40,9 @@ <h3>Pick tags for news</h3>
                                 [ngClass]="{
                                     'filters-color': filter.isActive
                                 }">{{ filter.name }}</button>
+                                <p [ngClass]="{
+                                    'red': isFilterValidation
+                                }">Only 3 tags can be added</p>
```

Comment:
please put tag`s ends (</button> and </p> in new line for better readability.
And put content inside tags also in new line

Reply:
fixed